tic_tac_toe.py

```diff
@@ -0,0 +1,227 @@
+import random
+
+x_symbol = "X"
+o_symbol = "O"
+player_1 = "Jackson"
+player_2 = "Super Mario"
+
+
+def create_board():  # Create the game board.
+    board = [["00", "01", "02"],
+             ["10", "11", "12"],
+             ["20", "21", "22"]]
+    return board
+
+
+def rules():  # Tic - Tac - Toe rules.
+    rules_open_message = """Welcome to my Tic-Tac-Toe game. Two players method. You need to do a strike of 3 symbols 
+    in line horizontal, vertical or diagonal. The most important thing is to do it faster than your enemy, 
+    other way you loss! If the whole board finished and there is no place to put your symbols while no one have 
+    strike of his own three symbol, it will be Tie.
+    Have Fun!"""
+    return rules_open_message
+
+
+def player_get_symbols():  # Each player get symbol.
+    players_symbols_dict = {player_1: x_symbol,
+                            player_2: o_symbol}
+    return players_symbols_dict
+
+
+def random_beginner():  # Randomize the symbols, every game the one who starts, start with "X" object.
+    # This function randomize the object that do the first move.
+    players_symbols_dict = player_get_symbols()
+    beginner = random.choice(list(players_symbols_dict.keys()))
+    second_player = ""
+
+    if beginner == player_1:
+        second_player = player_2
+    elif beginner == player_2:
+        second_player = player_1
+    random_players_symbols = {beginner: x_symbol,
+                              second_player: o_symbol}
+    return random_players_symbols
+
+
+random_beginner_var = random_beginner()
+
+
+def return_key():
+    # Get the key to know who start and who's th second one.
+    # It's not like the first dictionary because we randomize the objects.
+    # Need to know the winner by his symbol.
+    beginner = ""
+    second_player = ""
+    players_dict = random_beginner_var
+
+    count = 0
+    for key in players_dict:
+        count += 1
+        if count == 1:
+            beginner = key
+        if count == 2:
+            second_player = key
+    return [beginner, second_player]
```

Comment:
Use "choice" function. Read about it. This "choice" shall save you this unnecessary lines of code